RxJavaTemplate/app/src/main/java/co/nimblehq/rxjava/ui/screens/second/SecondFragment.kt

```diff
@@ -42,6 +42,11 @@ class SecondFragment : BaseFragment() {
             .addToDisposables()
     }
 
+    override fun handleVisualOverlaps() {
+        btOpenCamera.handleVisualOverlaps()
+        btOpenPost.handleVisualOverlaps()
```

Comment:
So how about `btSecondxxx`? 😅  And I think we could rename this screen to `Detail` instead. How do you think?

Reply:
@hoangnguyen92dn because the fragment name is `second` still so I don't make change on it, will not a big deal right?

Reply:
Yup 😅  We could leave it as it is for now.